src/core/utils/net.ts

```diff
@@ -157,3 +157,12 @@ export function hostnameToIpAdress(hostnameOrIpAddress: string | undefined) {
   }
   return hostnameOrIpAddress;
 }
+
```

Comment:
This function should be moved to `src/core/utils/cli.ts` and must be tested.

Reply:
Updated and added tests